Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -3608,6 +3684,31 @@ - (void)showErrorAlertForAttachment:(SignalAttachment * _Nullable)attachment {
                      completion:nil];
 }
 
+- (void)textViewDidChangeSize
+{
+    OWSAssert([NSThread isMainThread]);
+
+    BOOL wasAtBottom = [self isScrolledToBottom];
+    if (wasAtBottom) {
+        [self.scrollLaterTimer invalidate];
+        self.scrollLaterTimer = [NSTimer weakScheduledTimerWithTimeInterval:0.001f
```

Comment:
Can you explain the reason for not calling `scrollToBottomImmediately`  without delay in a code comment?